scheduler/src/cook/kubernetes/api.clj

```diff
@@ -719,41 +725,43 @@
   (let [api (CoreV1Api. api-client)
         ^V1DeleteOptions deleteOptions (-> (V1DeleteOptionsBuilder.) (.withPropagationPolicy "Background") .build)
         pod-name (-> pod .getMetadata .getName)
         pod-namespace (-> pod .getMetadata .getNamespace)]
     ; TODO: This likes to noisily throw NotFound multiple times as we delete away from kubernetes.
     ; I suspect our predicate of k8s-actual-state-equivalent needs tweaking.
-    (try
-      (.deleteNamespacedPod
-        api
-        pod-name
-        pod-namespace
-        deleteOptions
-        nil ; pretty
-        nil ; dryRun
-        nil ; gracePeriodSeconds
-        nil ; oprphanDependents
-        nil ; propagationPolicy
-        )
-      (catch JsonSyntaxException e
-        ; Silently gobble this exception.
-        ;
-        ; The java API can throw a a JsonSyntaxException parsing the kubernetes reply!
-        ; https://github.com/kubernetes-client/java/issues/252
-        ; https://github.com/kubernetes-client/java/issues/86
-        ;
-        ; They actually advise catching the exception and moving on!
-        ;
-        (log/info "Caught the https://github.com/kubernetes-client/java/issues/252 exception deleting" pod-name))
-      (catch ApiException e
-        (let [code (.getCode e)
-              already-deleted? (contains? #{404} code)]
-          (if already-deleted?
-            (log/info e "Pod" pod-name "was already deleted")
-            (throw e)))))))
+    (timers/time! (metrics/timer "delete-pod" compute-cluster-name)
+      (try
+        (.deleteNamespacedPod
+          api
+          pod-name
+          pod-namespace
+          deleteOptions
+          nil                                               ; pretty
+          nil                                               ; dryRun
+          nil                                               ; gracePeriodSeconds
+          nil                                               ; oprphanDependents
+          nil                                               ; propagationPolicy
+          )
+        (catch JsonSyntaxException e
+          ; Silently gobble this exception.
+          ;
+          ; The java API can throw a a JsonSyntaxException parsing the kubernetes reply!
+          ; https://github.com/kubernetes-client/java/issues/252
+          ; https://github.com/kubernetes-client/java/issues/86
+          ;
+          ; They actually advise catching the exception and moving on!
+          ;
+          (log/info "In" compute-cluster-name "caught the https://github.com/kubernetes-client/java/issues/252 exception deleting" pod-name))
+        (catch ApiException e
+          (meters/mark! (metrics/meter "delete-pod-errors" compute-cluster-name))
+          (let [code (.getCode e)
+                already-deleted? (contains? #{404} code)]
+            (if already-deleted?
+              (log/info e "In" compute-cluster-name "pod" pod-name "was already deleted")
```

Comment:
Same comment as above about the pattern for these logs.

Reply:
Changed